python/hail/keytable.py

```diff
@@ -60,30 +55,27 @@ def key_names(self):
         try:
             return list(self.jkt.keyNames())
         except Py4JJavaError as e:
-            self._raise_py4j_exception(e)
+            raise_py4j_exception(e)
 
     def column_names(self):
         """Names of all columns.
 
         :rtype: list of str
-
         """
         try:
             return list(self.jkt.fieldNames())
         except Py4JJavaError as e:
-            self._raise_py4j_exception(e)
+            raise_py4j_exception(e)
 
     def num_rows(self):
         """Number of rows.
 
         :rtype: long
-
         """
         try:
             return self.jkt.nRows()
         except Py4JJavaError as e:
-            self._raise_py4j_exception(e)
-        
+            raise_py4j_exception(e)
 
     def same(self, other):
         """Test whether two KeyTables are identical.
```

Comment:
You changed Variant Dataset to dataset (which I totally agree with).  Should we keep writing KeyTable here?